bin/scripts/config-helper.mjs

```diff
@@ -73,6 +73,15 @@ async function advancedConfigPrompts () {
         await askRoleQuestions();
     }
 
+    const policiesResponse = await prompt({
```

Comment:
I would just update `existingRoles` to say "use existing IAM Roles/Policies?". This won't be necessary for users not using Dynamic Roles, but I would just say it is optional and still group them together as it all is in the vein of IAM resources